modules/nf-core/kraken2/build/main.nf

```diff
@@ -0,0 +1,46 @@
+process KRAKEN2_BUILD {
+    tag "$meta.id"
+    label 'process_medium'
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-f8c4015c836dd3ce5c118cfed97ec8259bab9e9d:2e0b144854b4a3d69b5df7a0340a60db846cc8bf-0':
+        'biocontainers/mulled-v2-f8c4015c836dd3ce5c118cfed97ec8259bab9e9d:2e0b144854b4a3d69b5df7a0340a60db846cc8bf-0' }"
```

Comment:
While this matches KRAKEN2_ADD, it doesn't seem to match the main KRAKEN2_KRAKEN2 module, maybe it's worth bumping that (depending what the differences are?)

Reply:
The container in the `KRAKEN2_KRAKEN2` module failed with the `add` module (as far as I remember), maybe we could try the mulled container with the `KRAKEN2_KRAKEN2`  module and check if it works or not (?).

Reply:
Yes exactly - basically try to have them all in line

Reply:
Ok. But we can try that later I guess.